### DISCUSSION TITLE
Release typescript fix

### DISCUSSION THREAD

**Discussion by cdash04** [sensedeep/dynamodb-onetable#462] [General]
Can we release the typescript fix made 2 weeks ago by @Hibanka ? I'm still on 2.3.6 because of that 😢

**Comment by mobsense** [comment]
Which issue # are you referring to?  That will help.

**Comment by cdash04** [comment]
#458

**Comment by AdrianneManneyzPR** [comment]
Early release of TypeScript fixes is recommended to address fatal bugs or security vulnerabilities that may affect the code.

**Comment by mobsense** [comment]
Thank you. Will release later this week.
